support/bazel/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+cc_library(
+    name = "fmt",
+    srcs = [
+        "include/fmt/args.h",
+        "include/fmt/chrono.h",
+        "include/fmt/color.h",
+        "include/fmt/compile.h",
+        "include/fmt/core.h",
+        "include/fmt/format.h",
+        "include/fmt/format-inl.h",
+        "include/fmt/locale.h",
+        "include/fmt/os.h",
+        "include/fmt/ostream.h",
+        "include/fmt/printf.h",
+        "include/fmt/ranges.h",
+        "include/fmt/xchar.h",
```

Comment:
Shouldn't these be in hdrs?

Reply:
Yes you are right. Change it. Moved all include file to `hdrs`. Not sure if `format-inl.h` should be part of the public interface (maybe it makes sense to keep it in the `srcs` section .